RUFAS/biophysical/manure/digester/continuous_mix.py

```diff
@@ -0,0 +1,320 @@
+from RUFAS.rufas_time import RufasTime
+from RUFAS.units import MeasurementUnits
+from RUFAS.general_constants import GeneralConstants
+
+
+"""Volumetric ratio of carbon dioxide to methane generated during anaerobic digestion
+    (m^3 carbon dioxide / m^3 methane)."""
+CARBON_DIOXIDE_TO_METHANE_RATIO: float = 4 / 6
+
+"""Factor by which total ammoniacal nitrogen content is increased by the anaerobic digestion process (unitless)."""
+TAN_INCREASE_FACTOR = 1.60
+
+"""Value of R in the ideal gas law (L·atm/(mol·K)."""
+IDEAL_GAS_LAW_R = 0.0821
+
+"""Molar mass of methane (g)."""
+METHANE_MOLAR_MASS = 16.04
+
+"""Molar mass of carbon dioxide (g)."""
+CARBON_DIOXIDE_MOLAR_MASS = 44.01
+
+
+class ContinuousMix(Digester):
+    """
+    Defines the behaviors and attributes of an anaerobic digester type, specifically a continuous stirred tank reactor.
+
+    Parameters
+    ----------
+    name : str
+        Unique identifier of the anaerobic digester.
+    temperature_set_point : float
+        Temperature set point for the anaerobic digestion (°C).
+    hydraulic_retention_time : int
+        Number of days manure spends in the anaerobic digester (days).
+    biogas_leakage_fraction : float
+        Fraction of biogas generated in the anaerobic digester that escapes to the atmosphere through unintended
+        leakage and is not collected by the gas capture system (unitless).
+
+    Attributes
+    ----------
+    _manure_in_digester : ManureStream
+        The total amount of manure received by an anaerobic digester in a single day.
+    _temperature_set_point : float
+        Temperature set point for the anaerobic digestion (°C).
+    _hydraulic_retention_time : int
+        Number of days manure spends in the anaerobic digester (days).
+    _biogas_leakage_fraction : float
+        Fraction of methane generated in the anaerobic digester that escapes to the atmosphere through unintended
+        leakage and is not collected by the gas capture system (unitless).
+
+    """
+
+    def __init__(
+        self,
+        name: str,
+        temperature_set_point: float,
+        hydraulic_retention_time: int,
+        biogas_leakage_fraction: float,
+    ) -> None:
+        super().__init__(name=name, is_housing_emissions_calculator=False)
+        self._manure_in_digester: ManureStream = ManureStream.make_empty_manure_stream()
+        self._temperature_set_point: float = temperature_set_point
+        self._hydraulic_retention_time: int = hydraulic_retention_time
+        self._biogas_leakage_fraction: float = biogas_leakage_fraction
+
+    def receive_manure(self, manure: ManureStream) -> None:
+        """Receives and stores manure to be digested."""
+        is_received_manure_valid = self.check_manure_stream_compatibility(manure)
+        if is_received_manure_valid is False:
+            raise ValueError(f"Anaerobic digester {self.name} received an invalid manure stream.")
```

Comment:
```suggestion
            raise ValueError(f"Continuous mix digester {self.name} received an invalid manure stream.")
```
Not sure if this is supposed to be changed to match the new file/class name?